scripts/addons.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Odoo, Open Source Management Solution
+#    Copyright (c) 2010-2015 Elico Corp.
+#    Augustin Cisterne-Kaas <augustin.cisterne-kaas@elico-corp.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import os
+from subprocess import call
+
+DOWNLOAD_PATH = '/mnt/data/additional_addons/'
+ODOO_CONF = '/opt/odoo/etc/odoo.conf'
+addons_path = ['/opt/odoo/sources/odoo/addons']
+
+
```

Comment:
As a general rule, what happens if the oca_dependencies.txt file is not correct? If the lines are malformed or if the repository cannot be downloaded because the project/branch doesn't exist for instance. We should make sure the instance won't start and an error is raised. Can be planned for a future version :)